Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -328,12 +329,12 @@ extension GoogleSheetsManager {
 
 // MARK: Writing to Google Sheets
 extension GoogleSheetsManager {
-  func addTransaction(amount: String, date: Date, category: Int, account: Int, transactionType: Int, approvalType: Int, completion: @escaping (Bool) -> Void) {
+  func addTransaction(amount: String, memo: String, date: Date, category: Int, account: Int, transactionType: Int, approvalType: Int, completion: @escaping (Bool) -> Void) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/ca9f87ec120f1846bdadbae33cf063c210ffd1f2/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L332"></span>Function should have 5 parameters or less: it currently has 8<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:332</code></td>
    </tr>
  </tbody>
</table>